dist/index.d.ts

```diff
@@ -428,6 +444,23 @@ declare global {
     stencilWriteMask?: number;
   }
 
+  export interface GPUDepthStencilState {
+    format: GPUTextureFormat;
+
+    depthWriteEnabled?: boolean;
+    depthCompare?: GPUCompareFunction;
+
+    stencilFront?: GPUStencilStateFaceDescriptor;
+    stencilBack?: GPUStencilStateFaceDescriptor;
```

Comment:
These should be updated to `GPUStencilFaceState`

Reply:
Thanks for catching that! Done.